shimmingtoolbox/cli/mask.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python3
+
+import click
+import nibabel as nib
+import numpy as np
+import os
+
+import shimmingtoolbox.masking.threshold
+from shimmingtoolbox.masking.shapes import shape_square
+from shimmingtoolbox.masking.shapes import shape_cube
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.group(context_settings=CONTEXT_SETTINGS, help=f"Creates a cube, square or threshold mask.")
+def mask():
+    pass
+
+
+@mask.command(context_settings=CONTEXT_SETTINGS,
+              help=f"Creates a cube mask from the input file. Returns an output nifti file with cube mask.")
+@click.option('-input', 'fname_input', type=click.Path(), required=True, help="Complete input path of the nifti file to"
+                                                                              " mask")
+@click.option('-output', type=click.Path(), default=os.curdir, help="Output folder for mask in nifti file")
+@click.option("-len_dim1", required=True, type=int, help="Length of the side of the cube along first dimension (in "
+                                                         "pixels)")
+@click.option("-len_dim2", required=True, type=int, help="Length of the side of the cube along second dimension (in "
+                                                         "pixels)")
+@click.option("-len_dim3", required=True, type=int, help="Length of the side of the cube along third dimension (in "
+                                                         "pixels)")
```

Comment:
in a cube, the three dimensions are the same, hence you only one flag

Reply:
same comment as in https://github.com/shimming-toolbox/shimming-toolbox/pull/137/files#r519030186, lighter code and more intuitive to have (replacing len by size):
~~~
st_mask size x
~~~

also: avoid mixing - and _

Reply:
Is it the same principle for the square mask or do we have to consider 2 sizes for this one?

Reply:
same principle